components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -163,7 +175,11 @@ - (instancetype)initWithOriginalPresentingViewController:
     _maximumInitialDrawerHeight =
         self.presentingViewBounds.size.height * kInitialDrawerHeightFactor;
     _shouldPresentAtFullscreen = NO;
-    _headerShadowColor = [[UIColor blackColor] colorWithAlphaComponent:(CGFloat)0.2];
+    UIColor *shadowColor = [[UIColor blackColor] colorWithAlphaComponent:(CGFloat)0.2];
```

Comment:
Nit: `UIColor.blackColor` is a property. Please use dot syntax to access it.

Reply:
Done.